controllers/base.go

```diff
@@ -83,3 +107,20 @@ func (c *BaseController) ensureGuestUser() bool {
 
 	return currentUser == nil
 }
+
+func (c *BaseController) applyCustomLayout() {
+	c.LayoutSections = make(map[string]string)
+	c.LayoutSections["FlashMessage"] = "shared/alert.html"
+	c.LayoutSections["HeaderContent"] = "shared/header.html"
+}
+
+func (c *BaseController) initActionPolicy() {
+	c.actionPolicy = make(map[string]Policy)
+}
+
+func (c *BaseController) updateCurrentUserData() {
```

Comment:
Not sure if it's an update 🤔 Updating would mean to change a value from something to something else. How about using set or assign here? Assign can be better in the sense that it's used in the view 💭 

```suggestion
func (c *BaseController) assignCurrentUser() {
```

Reply:
updated here https://github.com/Lahphim/go-crawler-challenge/pull/51/commits/ec0ed00ac14f5ca9bcd6cde4d8b644d5f287c5d0